src/components/landing-page/create-production.tsx

```diff
@@ -178,10 +229,36 @@ export const CreateProduction = () => {
           </PrimaryButton>
         </ButtonWrapper>
       </FlexContainer>
-      {createdProductionId !== null && (
-        <ProductionConfirmation>
-          The production ID is: {createdProductionId.toString()}
-        </ProductionConfirmation>
+      {createdProductionId !== null && production && (
+        <>
+          <ProductionConfirmation>
+            The production ID is: {createdProductionId.toString()}
+          </ProductionConfirmation>
```

Comment:
I don't think the production confirmation message should be dependent on the `production` successfully having been fetched, since the created production id is returned in the create production response.